src/java.instrument/share/classes/java/lang/instrument/ClassFileTransformer.java

```diff
@@ -160,6 +160,20 @@
  * sequence of bytes in class file format, whether or not they reside in a
  * file.
  *
+ * @apiNote
+ * If the invocation of {@link #transform transform()} for a class <code>C</code>
+ * directly or transitively requires loading or resolving the same class <code>C</code>,
+ * an error is thrown that is an instance of {@link LinkageError} (or a subclass).
+ * Transforming core JDK classes or using libraries which depend on core JDK classes
+ * during transformation increases the risk for such errors. If the {@link LinkageError}
+ * occurs during reference resolution (see section 5.4.3 Resolution of <cite>The
```

Comment:
```suggestion
 * occurs during reference resolution (see section {@jvms 5.4.3} Resolution of <cite>The
```
Other stylistic note: we can replace `<code>C</code>` with `{@code C}`, which is shorter.

Reply:
Thanks for your suggestion @liach . I didn't knew about the `@jvms` tag which is quite handy. I've also updated the other reference to the JVMS above the new `@apiNote` which didn't used the tag either (and was wrong because the JVMS sections have changed since Java 5 :)